Assets/MixedRealityToolkit.Extensions/HandPhysicsService/Microsoft.MixedReality.Toolkit.Extensions.HandPhysics.asmdef

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Microsoft.MixedReality.Toolkit.Extensions.HandPhysics",
+    "references": [
+        "Microsoft.MixedReality.Toolkit",
+        "Microsoft.MixedReality.Toolkit.Services.InputSystem",
```

Comment:
does this need a hard-link to a specific input system implementation? doing so limits flexibility (cannot use with a custom implementation)